runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/config/NamspaceRefAdapter.java

```diff
@@ -27,12 +27,21 @@
 import jakarta.json.JsonValue;
 import jakarta.json.bind.adapter.JsonbAdapter;
 
+import io.aklivity.zilla.runtime.engine.config.ConfigAdapterContext;
+
 public class NamspaceRefAdapter implements JsonbAdapter<NamespaceRef, JsonObject>
 {
     private static final String NAME_NAME = "name";
     private static final String LINKS_NAME = "links";
 
     private static final Map<String, String> LINKS_DEFAULT = emptyMap();
+    private final ConfigAdapterContext context;
```

Comment:
Please put a blank line above `context` to keep it separate from static fields for better readability.